jstwonon

```diff
@@ -0,0 +1,25 @@
+ublic class GCDExample1 {
```

Comment:
Kindly refactor the code by referring to the pr. 
https://github.com/TheAlgorithms/Java/blob/master/Maths/Area.java